### PR TITLE
init runtime details for coprocessor

### PR DESCRIPTION
**Pull request by AndreMouche**
@hicqu @shenli @ilovesoup PTAL

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -46,6 +46,30 @@ message Context {
     IsolationLevel isolation_level = 7;
     bool not_fill_cache = 8;
     bool sync_log = 9;
+    bool handle_time = 10;
+    bool scan_detail = 11;
+}
+
+message HandleTime {
+    int64 wait_ms = 1;
+    int64 process_ms = 2;
```
Comment (hicqu) [inline proto/kvrpcpb.proto]: Does `process_ms` contains `wiat_ms`?

Comment (AndreMouche) [reply proto/kvrpcpb.proto]: no @hicqu

**Comment by AndreMouche** [comment]
@hicqu @shenli @ilovesoup PTAL

**Comment by AndreMouche** [comment]
@hicqu @shenli @ilovesoup PTAL

**Comment by shenli** [comment]
LGTM

**Comment by hicqu** [comment]
LGTM.

**Pull request merged**